service-tests/github.js

```diff
@@ -205,3 +205,87 @@ t.create('hit counter for nonexistent repo')
     name: Joi.equal('goto counter'),
     value: Joi.string().regex(/^repo not found$/),
   }));
+
+t.create('open issues')
+  .get('/issues/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? open$/)
+  }));
+
+t.create('open issues (raw)')
+  .get('/issues-raw/badges/shields.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('issues'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]?$/)
+  }));
+
+t.create('open pull requests')
+  .get('/issues-pr/cdnjs/cdnjs.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('pull requests'),
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]? open$/)
+  }));
```

Comment:
Should this just be `/pull-requests` instead of `/issues-pr`?

Reply:
Ah, we need to support `/issues-pr` for backward compatibility. What do you think about supporting both?

Reply:
Yup, so I only simplified the implementation and made the numbers exact (rather than saying 30+ for everything larger than 30). And that's what we needed. The simplification was simple a result of the fact that it was needed for getting proper numbers.